### ISSUE TITLE
Prune mode is incompatible with -txindex

### ISSUE THREAD

**Issue by pcartwright81** [Veil-Project/veil#253]
Windows 10 wallet won't open and says "Prune mode is incompatible with -txindex"

![image](https://user-images.githubusercontent.com/2352477/50940259-d7f57f00-1445-11e9-8735-5e859972f390.png)

**Comment by pcartwright81** [comment]
If anyone else has this issue fix the key in the registry under Computer\HKEY_CURRENT_USER\Software\Veil\Veil-Qt
You will see bPrune

**Comment by ianGre** [comment]
You could take "txindex=1" out of your veil.conf, or 
turn off (in Options) Prune block storage. (Except that you can't, because it won't start.) or
edit your Registry with Regedit (as mentioned above). You might want to turn off Prune mode when you get it running.

Prune mode (keep only blocks interesting to your wallet) and
txindex (index all blocks) are certainly contradictory instructions.

**Comment by pcartwright81** [comment]
I do not have a viel.conf and there does not seem to be an option in the GUI to turn off index all blocks.

**Comment by ianGre** [comment]
Interesting! I'll give you some instructions to do the fix via the Windows Registry. Just a moment while I start up my Windows Virtual Machine. ...

**Comment by ianGre** [comment]
I am surprised that pruning is turned on, as it was off by default with my installation. Also I did not find a registry entry for transaction indexing. So, the easiest way (to turn off pruning, assuming your wallet will not open, because of the error), is to open Notepad and paste the following lines into it, and save the file onto your Desktop as "Veil-pruneOff.reg" or whatever you like. And then right-click and select **Merge**, which should be the default action, if the filename ends in .reg and not.txt. 
```
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\Veil\Veil-Qt]
"bPrune"="false"
```

**Comment by presstab** [comment]
txindex is turned on by default with the code. Interesting that prune is defaulting as on... it actually gets disabled in the code, as it is still under development.

**Comment by seanPhill** [comment]
I haven't seen prune on by default in any of my installations.

**Comment by seanPhill** [comment]
Thanks! ... Well, then the prune mode function had better turn off txindex when it turns on.

**Comment by mimirmim** [comment]
Update on this issue as we have been investigating why all nodes are flagging as limited nodes, and not actually full nodes. This is related.

**Comment by seanPhill** [comment]
This issue has been addressed with merge #874 (Ignore prune mode if txindex is set) -- not to say that prune mode works -- it doesn't, and has not been developed yet.

(cleaning up)
